modules/internal/DefaultRectangular.chpl

```diff
@@ -1340,33 +1343,32 @@ module DefaultRectangular {
                                                    BoundedRangeType.bounded,
                                                    stridable)) {
       on this {
-        const allocD = {(...bounds)};
+        const reallocD = {(...bounds)};
 
         // For now, we'll use realloc for 1D, non-empty arrays when
         // the low bounds and strides of the old and new domains
         // match; and when the element type is POD or the array is
         // growing (i.e.,(doesn't require deinit to be called).
-        if (rank == 1 &&
-            allocD.low == dom.dsiLow && allocD.stride == dom.dsiStride &&
-            dom.dsiNumIndices > 0 && allocD.size > 0 &&
-            (isPODType(eltType) || allocD.size > dom.dsiNumIndices)) {
+        if (!disableArrRealloc && rank == 1 &&
+            reallocD.low == dom.dsiLow && reallocD.stride == dom.dsiStride &&
+            dom.dsiNumIndices > 0 && reallocD.size > 0) {
+
           if reportInPlaceRealloc then
             writeln("reallocating in-place");
 
-          sizesPerDim(1) = allocD.dsiDim(1).size;
```

Comment:
Why is this line deleted? I don't see anything about it in the PR message.

Reply:
Huh, I suspect that was a mistake when I was renaming allocD to reallocD.  Good catch.